myTrip/static/src/comment/CommentItem.js

```diff
@@ -32,9 +38,11 @@ export class CommentItem extends React.Component {
 
     handleEditCommentText = (event) => {
         this.setState({'editCommentText': event.target.value});
-        if (this.state.editCommentText.length !== 0) {
-            this.setState({'disabled': false});
-        };
+        if ((event.target.value !== this.props.message) && (event.target.value != 0)) {
```

Comment:
No because in this case it means if one of conditions is true => allow submit, and we need to allow submit both conditions are true.